Changelog.md

```diff
@@ -31,6 +31,10 @@ Bugfixes:
  * Yul IR Generator: Do not output empty switches/if-bodies for empty contracts.
 
 
+Build System:
+ * Remove obsolete compatibility workaround for emscripten builds.
```

Comment:
Not sure we need this...

Reply:
Yeah, I was wondering whether to add it... but if unexpectedly anyone tries to use soljson.js without solc-js and expects this hack, maybe nice to add least have some mention of it...